website/src/modules/pages/auth/login/LoginPage.tsx

```diff
@@ -32,7 +41,15 @@ export const LoginPage: React.VFC = () => {
       }
 
       if (redirectUrl) await router.push(redirectUrl);
-      else if (window.opener) window.close();
+      else if (window.opener) {
+        // TODO: validate window opener origin
+        console.log('window.opener', window.opener);
+        const idToken = await currentUser().getIdToken();
+        // TODO: change origin
+        window.opener.postMessage(JSON.stringify({ type: 'LOGIN', content: { idToken } }), '*');
```

Comment:
Can be removed? Or maybe we will need some new kind of message, like `AUTH_COMPLETED` to refresh opener window?